Gruntfile.js

```diff
@@ -1,49 +1,48 @@
 'use strict';
 
 module.exports = function(grunt) {
-
   grunt.initConfig({
-    mocha_istanbul: {
+    'mocha_istanbul': {
       coverage: {
         src: ['unit_tests', 'src'],
         options: {
-          mask: '**/*.js'
-        }
-      }
+          mask: '**/*.js',
+        },
+      },
     },
-    istanbul_check_coverage: {
+    'istanbul_check_coverage': {
       default: {
         options: {
           coverageFolder: 'coverage*',
           check: {
             lines: 70,
-            statements: 70
-          }
-        }
-      }
+            statements: 70,
+          },
+        },
+      },
     },
-    jshint: {
-      files: ['Gruntfile.js', 'src/**/*.js', 'unit_test/**/*.js'],
+    'eslint': {
       options: {
-        globals: {
-          jQuery: true
+        configFile: '.eslintrc.yml',
+      },
+      target: ['Gruntfile.js', 'src/**/*.js', 'tests/**/*.js'],
+    },
+    'babel': {
+      options: {
+        sourceMap: true,
+      },
+      dist: {
+        files: {
+          'dist/index.js': 'src/index.js',
         },
-        ignores: ['src/node_modules/**'],
-        esversion: 6,
-        node: true
-      }
+      },
     },
-    watch: {
-      files: ['<%= jshint.files %>'],
-      tasks: ['jshint']
-    }
   });
 
   grunt.loadNpmTasks('grunt-mocha-istanbul');
-  grunt.loadNpmTasks('grunt-contrib-jshint');
-  grunt.loadNpmTasks('grunt-contrib-watch');
-
-  grunt.registerTask('default', ['jshint']);
-  grunt.registerTask('test', ['jshint', 'mocha_istanbul:coverage', 'istanbul_check_coverage']);
+  grunt.loadNpmTasks('grunt-eslint');
+  grunt.loadNpmTasks('grunt-babel');
 
+  grunt.registerTask('default', ['eslint', 'babel']);
```

Comment:
I would use another identifier for the task instead of default. What do you think about babelfy

Reply:
Also, there is no need to add it to the test task